lib/web/mage/utils/main.js

```diff
@@ -6,7 +6,10 @@ define(function (require) {
     'use strict';
 
     var utils = {},
-        _ = require('underscore');
+        _ = require('underscore'),
+        root = typeof self == 'object' && self.self === self && self ||
+            typeof global == 'object' && global.global === global && global || Function('return this')() || {};
+    root._ = _;
 
```

Comment:
should have new line after var declare

Reply:
@mrtuvn the latest underscore version doesn't add the `_` into the global variable `window` as it was doing before. And here, we're trying to push it to `root` variable, which is `window` in browsers, or might be different in different environments.

Reply:
Can it be moved to the tests?

Reply:
What exactly do you mean, by moving it to tests?
Basically, for tests it is used the `package.json.sample` file to install the needed dependencies.

Reply:
I mean adding underscore to the root, it should be only in tests, right? Or is it breaking some functionality in the functionality?

Reply:
No, this happens on the storefront/backend as well. In this file we address this issue, and the `package.json.sample` seems to fix it for the jasmine tests.

At the moment the Functional CE and EE are successfully passing, which means we are on the good way.